CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.42.0-rc.0 / 2023-01-26
```

Comment:
Hm, I am missing some changes worth mentioning. Could you double check?

e.g.

* Native histogram improvements:
   * https://github.com/prometheus/prometheus/pull/11864
  * Promtool: https://github.com/prometheus/prometheus/pull/11872

There might be more.

Reply:
I'll re-check the history